app/src/app/routes/CreateMapping.tsx

```diff
@@ -154,19 +159,67 @@ const CreateMapping = (): JSX.Element => {
         }).unwrap();
 
         try {
-          // Column creation
-          const createdColumns = await Promise.all(
+          // Filters Column creation
+          const filterColumns = await Promise.all(
             filters.map(({ col }) =>
               createColumn({
                 columnRequest: { ...col, owner: owner.id } as Column,
               }).unwrap()
             )
           );
 
+          // Filters Joins creation
+          const filtersJoins = await Promise.all(
```

Comment:
We create joins before filters. So if some joins fails, we get orphan joins without their filters no?

Reply:
Yes I guess there's not any reason we would create joins before filters. I'll change that